### ISSUE TITLE
Strange bug when using array as parameter for contract method.

### ISSUE THREAD

**Issue by kasperfish** [EOSIO/eosjs#198]
The contract is confirmed to work with cleos. The issue highly likley arise in the js lib. When using an array as input for a call on on our smart contract something strange happens. look for yourself.

**this works (one object in the array)**
![image](https://user-images.githubusercontent.com/5130772/41456885-24b59b44-7082-11e8-82ce-6ab5ab866e8c.png)

**this doesn't work (two objects in array)**
![image](https://user-images.githubusercontent.com/5130772/41456812-ee7fc784-7081-11e8-8233-36ca441e7b4a.png)

![image](https://user-images.githubusercontent.com/5130772/41456784-cf7548f0-7081-11e8-9760-2809fe45eff1.png)

Also get duplicate transaction error when running the script multiple times.

**Comment by jcalfee** [comment]
Probably related: https://github.com/EOSIO/eosjs/issues/189

**Comment by jcalfee** [comment]
The duplicate transaction error is normal.  Either change the transaction a little or include nonce action to make the transaction different (nonce was just added in the last release).

**Comment by kasperfish** [comment]
yes this is probably related... suddenly the number 4 makes sense.

**Comment by kasperfish** [comment]
Ok now it seems that this error also pops up in a normal EOS transfer.

![image](https://user-images.githubusercontent.com/5130772/41509137-95ebccd8-724f-11e8-993b-aaf792f85057.png)


![image](https://user-images.githubusercontent.com/5130772/41509128-7304a3ac-724f-11e8-9641-c6435ae82d5c.png)

**Comment by jcalfee** [comment]
You have to keep changing the transfer amount if your testing.  Are you saying it is rejecting after a delay or under normal production activity?  We can probably find a discussion at EOSIO/eos where this logic lives.  Just make sure your separating out your testing with production and make sure you alter your test transactions to get around this..

**Comment by snipper15** [comment]
quantity:'1.0000 4,EOS' means that you passed what quantity has 4 zero after the point ,
